Description: this patch changes the man page sections from 1L to 1.
             This is also already changed upstream but not released yet.
Author: Klaus Reimer <kay@debian.org>
Last-Update: 2011-01-01
Index: arc/arc.1
===================================================================
--- arc.orig/arc.1
+++ arc/arc.1
@@ -1,4 +1,4 @@
-.TH ARC 1L "11 Nov 1991" "Howard Chu@JPL" "LOCAL COMMANDS"
+.TH ARC 1 "11 Nov 1991" "Howard Chu@JPL" "LOCAL COMMANDS"
 .SH NAME
 arc \- pc archive utility
 .SH SYNOPSIS
Index: arc/marc.1
===================================================================
--- arc.orig/marc.1
+++ arc/marc.1
@@ -1,4 +1,4 @@
-.TH MARC 1L "25 Nov 1992" "Howard Chu@JPL" "LOCAL COMMANDS"
+.TH MARC 1 "25 Nov 1992" "Howard Chu@JPL" "LOCAL COMMANDS"
 .SH NAME
 marc \- archive merger
 .SH SYNOPSIS
